src/app/personsok/Sokeresultat.tsx

```diff
@@ -18,19 +18,24 @@ function Sokeresultat(props: Props) {
 
     const tittelRekke = ['Fødselsnummer', 'Navn', 'Adresser', 'Bosted'];
     const trengerUtenlandskIDTittel = props.response.some(
-        person => person.utenlandskID?.utenlandskID.identifikasjonsnummer !== undefined
+        person => person.utenlandskID?.identifikasjonsnummer !== undefined
```

Comment:
Kanskje man kunne endre navnet til noe mer generelt for begge sjekkene? `harUtenlandsID` eller `trengerUtenlandskIDKolonne`

Reply:
Endret det nå 😄